signac/contrib/project.py

```diff
@@ -170,9 +169,9 @@ def __str__(self):
         return str(self.id)
 
     def __repr__(self):
-        return "{type}.get_project('{root}')".format(
+        return "{type}.get_project({root})".format(
                    type=self.__class__.__name__,
-                   root=self.root_directory())
+                   root=repr(self.root_directory()))
```

Comment:
Change isn't really related to windows compatibility, but makes sense.

Reply:
This is not obvious, but the change is needed for Windows compatibility The `repr` is needed to make the backslashes work correctly in the path string. Note that the quotes were also removed (since the `repr` includes quotes).

Reply:
For a few crucial classes with minimal "state" like this one, we try to adhere to the invariant `eval(repr(obj)) == obj`. That is, we want a `repr` that can be copy-pasted into a Python command and used.

Reply:
Ah okay, thanks for informing me.